mantidproject/mantid
# RROA Remove summing of transmission run workspaces

**Description of work.**
Removed the summation of transmission workspaces, and made it always divide by the first transmission run opposed to the corresponding group member. Added extra tests to test for these circumstances
**Report to:** Jos

**To test:**
Run this script in MantidPlot:
``` python
OFFSPEC44956 = LoadISISNexus(Filename="OFFSPEC44956")
OFFSPEC44937 = LoadISISNexus(Filename="OFFSPEC44937")
ReflectometryReductionOneAuto(InputWorkspace='OFFSPEC44956', ThetaIn=0.7, 
  MomentumTransferStep=0.03, MomentumTransferMax=0.1, 
  OutputWorkspaceBinned='IvsQ_binned', OutputWorkspace='IvsQ', OutputWorkspaceWavelength='IvsLam',
  FirstTransmissionRun="OFFSPEC44937", AnalysisMode="MultiDetectorAnalysis",
  ProcessingInstructions="400-410")
```

Plot IvsQ_binned_1, then change it to a log(x),log(y) by right clicking and changing axes and it should look like this:
![image](https://user-images.githubusercontent.com/24723609/44590259-a2136f00-a7b2-11e8-9b1d-06425682b156.png)


Fixes #23086 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
